storage/src/main/java/org/apache/kafka/storage/internals/log/LogConfig.java

```diff
@@ -515,8 +515,8 @@ public Boolean remoteLogDeleteOnDisable() {
         return remoteLogConfig.remoteLogDeleteOnDisable;
     }
 
-    public Boolean remoteCopyDisabled() {
-        return remoteLogConfig.remoteCopyDisabled;
+    public Boolean remoteLogCopyDisable() {
```

Comment:
My suggestion was to change the name of the topic config. 

This method name sounds like a setter. can we rename it to `isRemoteLogCopyDisabled` here and other places?

(eg) https://github.com/apache/kafka/pull/16256

Reply:
I'm fine. But I'd like to make it as a separate PR. That's not urgent. Thanks.